.github/workflows/pr.yaml

```diff
@@ -291,6 +291,26 @@ jobs:
           args: --all-projects --org=product-engineering-ly9
         continue-on-error: true
 
+  notify-failure:
+    name: Send Slack alert
+    needs:
+      - ci-go
+      - ci-static
+      - ci-js
+      - ci-build-gitops-image
+      - ci-upload-images
+      - ci-upload-binary
```

Comment:
I think you can replace this list with just `finish-ci-merge`

Reply:
i did start with that, changed for... reasons i forget